### PR TITLE
Cleanup window function registry tests

### PR DESCRIPTION
**Pull request by pramodsatya**
There are two test files for validating window function registration:
1. velox/functions/prestosql/window/tests/WindowFunctionRegTest.cpp
2. velox/exec/tests/WindowFunctionRegistryTest.cpp

This PR moves the tests from 1 into 2, and removes 1. This follows from the
conversation in #7243.

### CODE REVIEW

File: velox/exec/tests/WindowFunctionRegistryTest.cpp [hunk 1]
```diff
@@ -17,7 +17,8 @@
 
 #include "velox/exec/WindowFunction.h"
 #include "velox/expression/SignatureBinder.h"
-#include "velox/type/Type.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
```
Comment (aditi-pandit) [inline velox/exec/tests/WindowFunctionRegistryTest.cpp:20]: Is this include file necessary ?

**Review by aditi-pandit — COMMENTED** [review]
Thanks @pramodsatya. Had one minor comment

File: velox/exec/tests/WindowFunctionRegistryTest.cpp (hunk 1 above)
Comment (pramodsatya) [reply velox/exec/tests/WindowFunctionRegistryTest.cpp]: Thanks for pointing it out, the include file is not needed and is removed now.

**Review by aditi-pandit — APPROVED** [review]
Thanks @pramodsatya

**Comment by aditi-pandit** [comment]
@mbasmanova : PTAL.

**Review by mbasmanova — APPROVED** [review]
@pramodsatya Thanks. Looks good to me.

@kagamiori Wei, would you help review and merge this PR?

**Review by kagamiori — APPROVED** [review]
LGTM. Thank you for cleaning up the code!

**Pull request closed without merging**